### ISSUE TITLE
Submit new version to CRAN before May 11

### ISSUE THREAD

**Issue by sfirke** [sfirke/janitor#113]
The current version of janitor is now slightly malfunctioning under upcoming dplyr release 0.6.0.  Hadley says he plans to submit to CRAN on May 11.  I've fixed the GitHub version of janitor to address dplyr 0.6.0, but at that point it becomes extremely problematic if the CRAN version of janitor is buggy with everyone's new dplyr.  So I must submit to CRAN before then.  And given my history of needing multiple tries to get accepted, I better leave some time.

I've fixed the dplyr-related bugs.  But as long as I'm submitting to CRAN I should make other fixes too.  I definitely need to close #103.  But should I go all-out on major changes and try to get a v 1.0 on CRAN ahead of dplyr?

I think the idea for breaking up `adorn_crosstab` in #101 is pretty set in my mind and could be implemented.  And replacing crosstab with `tabyl` taking two columns shouldn't be that hard.  Then close as many other issues as I can.

To do this right I probably need to have v.1.0 ready by April 28, then have people test it out for a week and submit to CRAN by May 5 after fixing any bugs.  It's gonna be tight timing...

**Comment by rgknight** [comment]
Let me know if you decide to target 1.0 by April 28th. We should probably hop on the phone to discuss which things I could / should own. Would definitely be tight, but without a deadline it might never happen.

**Comment by sfirke** [comment]
Alright, I fixed #103 and could probably submit to CRAN.  Certainly with that in the bag, the important thing is to precede dplyr, so that no one is getting bad results.  There are some other issues it would be nice to get to, but I don't think it'll happen.

I do get this warning from `check()`:
```
R CMD check results
0 errors | 1 warning  | 0 notes
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'add_totals_col' is deprecated.
  Warning: 'add_totals_row' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```
Maybe eliminating unit tests (from the CRAN version - I like them in the GitHub version as long as those functions exist) will silence that?  Or I could see if CRAN will excuse it, with an explanation.  Or I could wrap the tests to suppress the warnings, like this: http://stackoverflow.com/a/42952304/4470365

**Comment by sfirke** [comment]
submitted v 0.3.0 to CRAN 🤞
